pkg/cmd/cli/cmd/set/env.go

```diff
@@ -48,6 +48,12 @@ syntax.`
   # Update all containers in all replication controllers in the project to have ENV=prod
   %[1]s env rc --all ENV=prod
 
+  # Import environment from a secret
+  %[1]s env --from-secret=mysecret dc/myapp
```

Comment:
It's clever, but it's not necessarily obvious to end users.  It makes
some assumptions that an object can map to key value pairs AND be
applied to an object.

What if I want to copy the environment from one DC to another?  Do I use --from?